jenkins/opensearch-dashboards/integ-test-opensearch-dashboards.jenkinsfile

```diff
@@ -0,0 +1,83 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    parameters {
+        string(
+            defaultValue: '',
+            name: 'opensearch_version',
+            description: 'OpenSearch version to run test on',
+            trim: true
+        )
+
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_version',
+            description: 'OpenSearch Dashboards version to run test on',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_build_id',
+            description: 'The build number of OpenSearch for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_build_id',
+            description: 'The build number of OpenSearch Dashboards for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'architecture',
+            description: 'Architecture of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'platform',
+            description: 'Platform of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
+    }
+    stages {
+        stage('integ-test') {
+            agent {
+                docker {
+                    label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028'
+                    alwaysPull true
+                }
+            }
+            steps {
+                script {
+                    basePathOpenSearch = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${opensearch_version}/${opensearch_build_id}/${platform}/${architecture}"
```

Comment:
Could we use the BuildManifest class to derive these values?

Constantly building/rebuilding these paths is prone to error and doesn't handle future url modifications well.

Reply:
+1 to this

Reply:
Sure. Modified.